src/askui_runner/modules/core/infrastructure/agent/agent_file_service.py

```diff
@@ -0,0 +1,41 @@
+import os
```

Comment:
What do you think about moving this to `src/askui_runner/modules/agents/file_service`  renaming the class to `FileService` or `AgentsFileService` as this is only used there but not in core?